pom.xml

```diff
@@ -574,11 +586,27 @@
         <artifactId>scala-guice_${scala.version.prefix}</artifactId>
         <version>4.1.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.bdgenomics.adam</groupId>
+        <artifactId>adam-codegen_${scala.version.prefix}</artifactId>
+        <version>${project.version}</version>
+      </dependency>
     </dependencies>
   </dependencyManagement>
 
   <profiles>
     <profile>
+      <id>python</id>
```

Comment:
```
[INFO] --- maven-jar-plugin:3.0.1:jar (default-jar) @ adam-python-spark2_2.11 ---
[WARNING] JAR will be empty - no content was marked for inclusion!
[INFO] Building jar: adam/adam-python/target/adam-python-spark2_2.11-0.23.0-SNAPSHOT.jar
```
reminds me that the Sonatype maven release process with fail to validate a release if `-sources.jar` or `-javadocs.jar` are missing for any jar artifact.

How are we releasing the python API?  I assume our release process will not be using this profile, right?